cedar-policy-core/src/evaluator/err.rs

```diff
@@ -312,7 +320,7 @@ pub enum EvaluationErrorKind {
     /// Tried to get an attribute of a (non-entity) record, but that record
     /// didn't have that attribute
     #[error("record does not have the attribute `{0}`")]
-    #[diagnostic(help("available attributes: {1:?}"))]
+    #[diagnostic(help("available attributes: {}", if .1.len() == TOO_MANY_ATTRS { format!("{:?} (truncated, more attributes may exist)", .1) } else { format!("{:?}", .1) } ))]
```

Comment:
We can't make this the nice error message we have on `4` since this enum variant is public from `cedar-policy`